samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,139 @@
+* On the **P14** pin header, connect **RSET2** and **VSET2** pins with a jumper.
+
+Make the following connections on the DK:
+
+* Switch nRF power source **SW9** to **VDD**.
+* Switch VEXT -> nRF **SW10** to **ON**.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following Kconfig options to specify which testcase to run:
+
+.. TESTCASE_SET_BUCK_VOLTAGE:
+
+TESTCASE_SET_BUCK_VOLTAGE
+  This sample configuration enables simple output voltage testcase.
+  Voltage is set to 2.4 V (`NPMX_BUCK_VOLTAGE_2V4`) on BUCK 1.
+
+.. TESTCASE_OUTPUT_VOLTAGE:
+
+TESTCASE_OUTPUT_VOLTAGE
+  This sample configuration enables advanced output voltage testcase.
+  BUCK 1 is starting from voltage 1.0 V (`NPMX_BUCK_VOLTAGE_1V0`).
+  The output voltage will increase from the selected to the maximum voltage (3.3 V).
+
+.. TESTCASE_RETENTION_VOLTAGE:
+
+TESTCASE_RETENTION_VOLTAGE
+  This sample configuration enables retention voltage testcase.
+  GPIO1 (`NPMX_GPIO_INSTANCE_1`) is set to input mode and configured to be used as a retention input selector for BUCK 1 (`NPMX_BUCK_INSTANCE_1`).
+  Normal mode voltage is set to 1.7 V (`NPMX_BUCK_VOLTAGE_1V7`) and retention voltage to 3.3 V (`NPMX_BUCK_VOLTAGE_3V3`).
+  When you change the input state of the selected GPIO you should see voltage changes in the BUCK 1 output.
+
+.. TESTCASE_ENABLE_BUCKS_VIA_PIN:
+
+TESTCASE_ENABLE_BUCKS_VIA_PIN
+  GPIO3 (`NPMX_GPIO_INSTANCE_3`) is set input mode and configured to be used as enable mode selector for BUCK 1 (`NPMX_BUCK_INSTANCE_1`).
+  The output voltage of BUCK 1 is set to 3.3 V (`NPMX_BUCK_VOLTAGE_3V3`).
+  When you change the input state of the selected GPIO you should see voltage changes in the BUCK 1 output - from 0 V to previously selected voltage.
+
+.. note::
+
+  Only one Kconfig option may be enabled at a time.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/pmic/npmx/buck`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+|test_sample|
+
+To verify voltage, connect multimeter to VOUT1 (output voltage pin of BUCK1).
+Voltage can be changed only for BUCK1. BUCK2 voltage is set by default to 3.0 V.
+
+#. |connect_kit|
+#. |connect_terminal|
+#. For simple output voltage testcase: Check output voltage on EK VOUT1 - it should be 2.4 V.
+#. For advanced output voltage testcase: Check output voltage on EK VOUT1 - it should start at 1.0 V and increase every 200 ms up to 3.3 V.
+#. For retention voltage testcase: Connect EK GPIO1 to GND. Check output voltage on EK VOUT1 - it should be 1.7 V. Connect EK GPIO1 to VDD. Check output voltage on EK VOUT1 - it should be 3.3 V.
+#. For enable bucks via pin testcase: Connect EK GPIO3 to GND. Check output voltage on EK VOUT1 - it should be 0.0 V. Connect EK GPIO3 to VDD. Check output voltage on EK VOUT1 - it should be 3.3 V.
+
+Dependencies
+************
+
+This sample uses drivers from npmx.
```

Comment:
More that one, i.e., the BUCK one?

```suggestion
This sample uses the :file:`npmx_buck.h` npmx driver.
```

Reply:
I guess the header files are not drivers.

Reply:
Could be added when @b-gent confirms that npmx docs are linkable.